app/models/task.py

```diff
@@ -3,4 +3,25 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)  # task name
+    description = db.Column(db.String)  # task description
```

Comment:
Similar to `Goal`, do we want empty values for a task's title and description? Consider adding `nullable=False` to ensure that all tasks have a title and description (or not if an empty title/description is useful data to store)